podvm-builder/azure-podvm-image-cm.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
```

Comment:
Generally i think it would be easier if defaults values will be set in the script, so that:
1. for the default use case (pp?) it will work without the need in setting the additional CM
2. it will encourage normal users not to mess with parameters he don't need

and regardless to the above, we can generate CM templates for GPU case & CoCo case with different filenames (but the same Object name)

Reply:
also, have you considered using the same peer-pods-cm but with some prefixed values for image creation? maybe it will provide other complication thought, i didn't think it through..

Reply:
I would like to unify the podvm image creation for both pp and coco and use the same code, with config options. This will avoid maintenance headache as mostly the differences are minor (except shim and upstream commit used). 
I am using the configMaps as sort of a config file for the scripts. It comes with defaults that are set but one is free to change the defaults. Although I would like to add comments on what shouldn't be changed by regular users.

I discarded the use of existing peer-pods-cm because it's tied to CAA. This one is completely isolated and one can decide not to use it if public images are available or using some other mechanisms.